src/styles/index.scss

```diff
@@ -1,3 +1,117 @@
-body {
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap");
+$blackColor: #060b35;
+$greyColor: #616070;
+$blueColor: #00acdc;
```

Comment:
you can create a separate `variables.scss` file and store your colors there. (just import this file into index.scss using `@import`)